src/SerialFirmata.h

```diff
@@ -23,6 +23,10 @@
 #include <SoftwareSerial.h>
 #endif
 
+//uncomment SERIAL_STORE_AND_FORWARD to collect bytes received by serial port until the receive buffer 
+// gets filled or a data gap is detected to avoid forwarding single bytes at baud rates below 50000
+//#define SERIAL_STORE_AND_FORWARD
```

Comment:
That works as well, but prepend with `FIRMATA_`. I have been having issues with defines being overwritten by other libraries or even changes in the Arduino core since they are global. I have not been so good at following this myself but I need to get in the habit of it, or start using enums instead.